syntax = "proto3";

import "google/protobuf/empty.proto";
import "service/DataService.proto";
package apb.api.notification;

option java_package = "jp.co.apb.server.api.proto.notification";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.Notification";

message GetHeaderNotificationResponse {
  int32 gift_not_receive_count = 1;
  int32 friend_request_receive_count = 2;
  bool is_exist_unread_information = 3;
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service NotificationService {
  rpc GetHeaderNotification(.google.protobuf.Empty) returns (.apb.api.notification.GetHeaderNotificationResponse);
}

